gallery/src/styles/SelectableColors/SelectableColorsPage.tsx

```diff
@@ -0,0 +1,106 @@
+
+const customExampleCode =
+    [selectableColorCustomExampleCode, { content: selectableColorCustomExampleStyles, language: 'scss' }];
+
+const SelectableColorsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        RSC provides a collection of color values which are intended to be used anywhere that the user is intended to
+        be able to select from a small set of colors and anywhere that a collection of arbitrarily but distinctly
+        colored elements is needed. Examples of the former include tags and labels. Examples of the latter
+        include highlighted regions of a text document.
+      </NxP>
+      <NxP>
+        The colors that RSC provides for this purpose are known as the "selectable colors" and they are used within
+        the <NxCode>NxColorPicker</NxCode> and <NxCode>NxTag</NxCode> components as well as exposed publicly
+        for use in creating custom components.
+      </NxP>
+      <NxP>
+        For each named color within the selectable colors, two actual RGB color codes are provided, a lighter
+        variant and a darker one. This allows for two-tone components built with the selectable colors, as seen in
+        unselected selectable <NxCode>NxTag</NxCode>s.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>
+            Usage in <NxCode>NxColorPicker</NxCode> and <NxCode>NxTag</NxCode>
+          </NxH3>
+        </NxTile.SubsectionHeader>
+        <NxP>
+          As documented on its gallery page, <NxCode>NxColorPicker</NxCode> allows the user to select from a set
+          of colors. The colors available are precisely the "selectable colors" documented here. On the other side
+          of things, <NxCode>NxTag</NxCode> can receive a "selectable color" name as one of its props and will be
+          styled using that color.
+        </NxP>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>
+            Custom Usage of the Selectable Colors
+          </NxH3>
+        </NxTile.SubsectionHeader>
+        <NxP>
+          A number of CSS classes and custom CSS variables are provided to facilitate the use of the selectable
+          colors for styling custom components. Essentially, for each selectable color, a CSS class is provided
+          which defines two custom properties: <NxCode>--nx-selectable-color-light</NxCode>{' '}
+          and <NxCode>--nx-selectable-color-dark</NxCode>. Where a custom component wants to use a selectable
+          color, they would add that color's class to the element in question (or a parent) and then use the custom
+          properties to set styles as they see fit.  For instance, they might
+          use <NxCode>--nx-selectable-color-light</NxCode> to set the element's background while
+          using <NxCode>--nx-selectable-color-dark</NxCode> to set the text color.
```

Comment:
I'm not sure that we should say this unless we're sure that these light/dark colour pairs also pass accessibility requirements for contrast (@dturner32)

Reply:
> We shouldn't say this unless we've confirmed that these light/dark colour pairs also pass accessibility requirements for contrast

Ah yeah I'm pretty certain they don't pass contrast requirements.  Changed to suggest background and border rather than background and text. 45dede15